### PR TITLE
use uniform `itow` naming across packets

### PR DESCRIPTION
**Pull request by andrei-ng**

### CODE REVIEW

**Comment by andrei-ng** [comment]
@gwbres would you have time to approve this ?

**Comment by gwbres** [comment]
> @gwbres would you have time to approve this ?

Hello, yes I do.
what do you think about renaming to `tow` ?

**Comment by andrei-ng** [comment]
> > @gwbres would you have time to approve this ?
> 
> Hello, yes I do. what do you think about renaming to `tow` ?

IMO it should stay "itow" as in the manual since the meaning apparently (I had to search for it myself) is "Integer Time Of Week" which is meant to differentiate it from "TOW" that has typical units seconds.

What is your point of view?

**Comment by gwbres** [comment]
> > > @gwbres would you have time to approve this ?
> > 
> > 
> > Hello, yes I do. what do you think about renaming to `tow` ?
> 
> IMO it should stay "itow" as in the manual since the meaning apparently (I had to search for it myself) is "Integer Time Of Week" which is meant to differentiate it from "TOW" that has typical units seconds.
> 
> What is your point of view?

Oh I did'nt know it was the official name in the UBX definitions. Let's keep it that way then

**Review by gwbres — APPROVED** [review]

File: ublox/src/ubx_packets/packets.rs [hunk 1]
```diff
@@ -721,7 +721,7 @@ struct NavSat {
 struct NavOdo {
     version: u8,
     reserved: [u8; 3],
-    i_tow: u32,
+    itow: u32,
```
[the remark this answers is not in the document]
Comment (andrei-ng) [reply ublox/src/ubx_packets/packets.rs:724]: @gwbres Strange, I didn't see this message 4 days ago. Sorry, it escaped me.

**Comment by andrei-ng** [comment]
> > > > @gwbres would you have time to approve this ?
> > > 
> > > 
> > > Hello, yes I do. what do you think about renaming to `tow` ?
> > 
> > 
> > IMO it should stay "itow" as in the manual since the meaning apparently (I had to search for it myself) is "Integer Time Of Week" which is meant to differentiate it from "TOW" that has typical units seconds.
> > What is your point of view?
> 
> Oh I did'nt know it was the official name in the UBX definitions. Let's keep it that way then

I didn't know why it was named iTow specifically either before you asked. So good you asked, we both learned something today :) . 

For completeness, from the uBLOX F9P manual : 
> The original designers of GPS chose to express time/date as an integer week number (starting
with the first full week in January 1980) and a time of week (often abbreviated to TOW)
expressed in seconds. Manipulating time/date in this form is far easier for digital systems than
the more conventional year/month/day, hour/minute/second representation. Consequently,
most GNSS receivers use this representation internally, only converting to a more conventional
form at external interfaces. The iTOW field is the most obvious externally visible consequence
of this internal representation.

https://content.u-blox.com/sites/default/files/ZED-F9P_IntegrationManual_UBX-18010802.pdf

**Pull request merged**